omod/src/main/java/org/openmrs/module/fhir/providers/RestfulConditionResourceProvider.java

```diff
@@ -79,9 +79,10 @@ public Condition getResourceByObsUID(@IdParam IdType theId) {
 	}
 
 	/**
-	 * Search appointments by unique id
+	 * Search conditions by patient uuid
 	 *
 	 * @param patient object containing the patient details
+	 * @return Returns
 	 */
 	@Search
 	public List<Condition> findConditionssByPatient(
```

Comment:
Worth fixing this typo maybe? Even though it was there before this PR.

```suggestion
	public List<Condition> findConditionsByPatient(
```